src/interface/TWNEOSigner.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright © 2017-2020 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include <TrustWalletCore/TWNEOSigner.h>
+
+#include "../NEO/Signer.h"
+#include "../proto/NEO.pb.h"
+#include "../HexCoding.h"
+
+using namespace TW;
+using namespace TW::NEO;
+
+TW_NEO_Proto_SigningOutput TWNEOSignerSign(TW_NEO_Proto_SigningInput data) {
```

Comment:
see my last comment about signer, let's move this logic into `Signer` to keep TWNEOSigner as simple as possible, https://github.com/trustwallet/wallet-core/blob/master/codegen/lib/templates/newcoin/TWSigner.cpp.erb

Reply:
done